src/collections/members/gopi-vaibhav/index.mdx

```diff
@@ -0,0 +1,16 @@
+---
+name: Gopi Vaibhav
+position: Contributor
+image_path: ./gopi-vaibhav.webp
+github: gopivaibhav
+twitter: gopi_vaibhav
+linkedin: gopivaibhav
+layer5: 85339de7-8ca2-4543-8793-1ebb1e76bf99
+location: Lucknow, India
+bio: "I am a Final year CSE student from IIIT Lucknow. I am an experienced MERN Stack Developer with a solid background in Python web frameworks, notably Django. My journey in the tech world has led me to actively contribute to prominent open-source projects like Layer5 and Oppia, through which I started learning to use Golang, Gatsby, GraphQL, Docker, k8s, etc. Aside from coding, I am a passionate student who is continuously pursuing new hobbies, such as trying new video games and participating in outdoor activities. I am constantly open to new and intriguing possibilities and look forward to meeting like-minded professionals. If you have a passion for web development, open-source contributions, or technology in general, let's connect and discuss how we might collaborate to create extraordinary digital solutions! 🚀"
+badges: 
+  - Community
```

Comment:
It should be in lowercase `community` as it is not showing the profile when community tag is applied as a filter